app/javascript/mastodon/initial_state.js

```diff
@@ -113,8 +113,10 @@ export const expandSpoilers = getMeta('expand_spoilers');
 export const forceSingleColumn = !getMeta('advanced_layout');
 export const limitedFederationMode = getMeta('limited_federation_mode');
 export const mascot = getMeta('mascot');
+export const maxReactions = (initialState && initialState.max_reactions) || 1;
 export const me = getMeta('me');
 export const movedToAccountId = getMeta('moved_to_account_id');
+export const visibleReactions = getMeta('visible_reactions');
```

Comment:
Same as with glitch's initial_state.
Adding `visible_reactions` to `InitialStateMeta` typedef stops VSCode from complaining.
(Not required)